gtsfm/bundle/bundle_adjustment.py

```diff
@@ -250,12 +259,12 @@ def evaluate(
                 metric_name = "Filtered tracks triangulated with GT cams: {}".format(exit_code.name)
                 ba_metrics.add_metric(GtsfmMetric(name=metric_name, data=count))
 
-        ba_metrics.add_metrics(metrics_utils.get_stats_for_sfmdata(aligned_filtered_data, suffix="_filtered"))
-        # ba_metrics.save_to_json(os.path.join(METRICS_PATH, "bundle_adjustment_metrics.json"))
+            ba_metrics.add_metrics(metrics_utils.get_stats_for_sfmdata(aligned_filtered_data, suffix="_filtered"))
```

Comment:
`aligned_filtered_data` cannot exist if `cameras_gt` is None, so these lines need to be indented.

Reply:
marking as resolved, this small change was landed in another PR